lib/wasix/src/state/env.rs

```diff
@@ -668,17 +561,30 @@ impl WasiEnv {
             func_env
                 .data(&store)
                 .blocking_on_exit(Some(Errno::Noexec.into()));
-            return Err(err.into());
+            return Err(WasiThreadError::ExportError(err));
         }
 
         // If this module exports an _initialize function, run that first.
         if call_initialize {
+            if let Ok(apply_data_relocs) = instance.exports.get_function("__wasm_apply_data_relocs")
```

Comment:
Maybe a comment why this exists and what it's for?

Maybe just a link to https://github.com/WebAssembly/tool-conventions/blob/main/DynamicLinking.md ?

Reply:
Docs are coming...